src/Store/store.js

```diff
@@ -4,9 +4,16 @@ import thunk from 'redux-thunk';
 
 const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
 
+const logger = () => {
+  return (next) => (action) => {
+    console.log('dispatch - ', action);
+    return next(action);
+  };
+}
+
```

Comment:
Ideally you should keep index.js file as much clean as possible. So all the middlewares and utilities are to be stored in a separate files, possibly - even in a separate folder like 'utils' or whatever.